### PR TITLE
Setup coveralls and semaphore

### PR DESCRIPTION
**Pull request by ESteinman**
### User story
```
As a Programmer
I would like to see how much of my code that are covered in tests
I want to have both Coveralls and Semaphore installed.
```
### What have we learned
How to setup Coveralls
Found the semaphoreci.com
Learned more about git reset --hard

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 require 'cucumber/rails'
+require 'coveralls'
+Coveralls.wear_merged!('rails')
```
Comment (tochman) [inline features/support/env.rb]: This bredd to be on top of the file IMO. Double check this please

File: coverage/.resultset.json [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+        0,
+        0,
+        0,
+        0,
+        0
+      ]
+    },
+    "timestamp": 1530623731
+  }
+}
```
Comment (tochman) [inline coverage/.resultset.json]: Pls remove this from version control

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -1,4 +1,7 @@
 require 'spec_helper'
+require 'coveralls'
+Coveralls.wear_merged!('rails')
+
```
Comment (tochman) [inline spec/rails_helper.rb]: This needs to be on top of the file

**Review by tochman — CHANGES_REQUESTED** [review]

File: features/support/env.rb (hunk 1 above)
Comment (ESteinman) [reply features/support/env.rb]: Sure

File: coverage/.resultset.json (hunk 2 above)
Comment (ESteinman) [reply coverage/.resultset.json]: Will do

File: spec/rails_helper.rb (hunk 3 above)
Comment (ESteinman) [reply spec/rails_helper.rb]: Will do

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**